index.html

```diff
@@ -767,18 +850,155 @@ <h2><dfn>CaptureController</dfn></h2>
             [Exposed=Window, SecureContext]
             interface CaptureController {
               constructor();
-              // TODO: Add setFocusBehavior() in a separate PR.
+              undefined setFocusBehavior(CaptureStartFocusBehavior focusBehavior);
             };
           </pre>
           <dl data-link-for="CaptureController" data-dfn-for="CaptureController" class="methods">
             <dt>
               <dfn>constructor</dfn>
             </dt>
+            <dd>
+              Create a new object with the following internal slots:
```

Comment:
I like this simpler format, but we need to mention the type
```suggestion
              Create a new {{CaptureController}} object with the following internal slots:
```